logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -198,7 +198,44 @@ actual class CallManagerImpl(
             callingLogger.d("$TAG - onCallingMessageReceived")
         }
 
-    override suspend fun startCall(conversationId: ConversationId, callType: CallType, conversationType: ConversationType, isAudioCbr: Boolean) {
+    private fun sendHandlerSuccess(
+        messageString: String?,
+        conversationId: ConversationId,
+        avsSelfUserId: UserId,
+        avsSelfClientId: ClientId
+    ) {
+        scope.launch {
+            messageString?.let { message ->
+                withCalling {
+                    when (sendCallingMessage(conversationId, avsSelfUserId, avsSelfClientId, message)) {
+                        is Either.Right -> {
+                            wcall_resp(
+                                inst = deferredHandle.await(),
+                                status = 200,
+                                reason = "",
+                                arg = null
+                            )
+                        }
+                        is Either.Left -> {
+                            wcall_resp(
+                                inst = deferredHandle.await(),
+                                status = 400,
+                                reason = "Couldn't send Calling Message",
+                                arg = null
```

Comment:
We need to pass the correct error code here, it's not always 400.
and same thing for the arg, a value is needed

Reply:
Added a TODO so we can handle the errorCode later from `CoreFailure`